packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -3,11 +3,19 @@ pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/ownership/OwnableMixin.sol";
 import "@synthetixio/core-contracts/contracts/proxy/UUPSProxy.sol";
+import "@synthetixio/core-contracts/contracts/errors/AddressError.sol";
 import "../interfaces/IElectionModule.sol";
 import "../token/MemberToken.sol";
 import "../storage/ElectionStorage.sol";
 
 contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
+    error MemberTokenAlreadyCreated();
```

Comment:
You should only declare errors and events in one place; either the interface or the implementation.

A very recent decision was made to do so in the implementation.